modules/nwtc-library/src/VTK.f90

```diff
@@ -360,10 +356,7 @@ SUBROUTINE WrVTK_SP_header( FileName, descr, Un, ErrStat, ErrMsg )
       INTEGER(IntKi)  , INTENT(  OUT)        :: ErrStat              !< error level/status of OpenFOutFile operation
       CHARACTER(*)    , INTENT(  OUT)        :: ErrMsg               !< message when error occurs
    
-      !$OMP critical
-      CALL GetNewUnit( Un, ErrStat, ErrMsg )      
```

Comment:
For backward compatibility, could you wrap this call to `GetNewUnit` (here and the other places it's now called) in an `IF` statement like `IF (Un < 0) call GetNewUnit()`?

Also note that the `ErrStat` and `ErrMsg` variables from the calls to `GetNewUnit` are now ignored.